src/wrap-logger.ts

```diff
@@ -50,9 +50,4 @@ export interface LoggerWithTarget extends Logger {
   fatal: LoggerWithTarget
 }
 
-export interface ChildArgs {
-  options?: object
-  name?: string
-  id?: string | number | string[]
-  installation?: string
-}
+export type ChildArgs = { [key: string]: any }
```

Comment:
Could you leave the known options and just add `[key: string]: any` below line 57 to it?

```suggestion
export type ChildArgs {
  options?: object
  name?: string
  id?: string | number | string[]
  installation?: string
  [key: string]: any
}
```